apps/catalogue/src/views/TableMappingsView.vue

```diff
@@ -77,7 +77,7 @@
         <th><h6>Syntax</h6></th>
         <th><h6>Notes</h6></th>
       </tr>
-      <tr v-for="m in variablemappings">
+      <tr v-for="m in variablemappings" :key="JSON.stringify(m)">
```

Comment:
why not use index a key ?

Reply:
I suppose that data might change for same index?

Reply:
?? data might change for same index , if variablemappings changes the view gets updated , the key is not needed for reactivity

Reply:
Ok, updated the code